Functions/Quantification/xASL_quant_FSL.m

```diff
@@ -386,13 +401,58 @@
 
 		% For Time-encoded, we skip the first volume per block
 		if x.modules.asl.bTimeEncoded
```

Comment:
Can we already remove `if bTimeEncoded`? Because inside `if bMergingSessions` you have another `if bTimeEncoded`

Reply:
It will indeed be removed. But simplifying would be double work. Cause not only this code will be removed from here. But we will not even really move it anywhere as the corresponding code is already now in xASL_wrp_Resample. And what we add to the merging function will not use this code. 
So no point in simplifying cause we won't be using it at all in the future.

Reply:
OK